aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -199,7 +200,27 @@ In the following *appsettings.json* example:
 }
 ```
 
-An alternative to using `Path` and `Password` for any certificate node is to specify the certificate using certificate store fields. For example, the `Certificates` > `Default` certificate can be specified as:
+Schema notes:
+
+* Endpoints names are case-insensitive. For example, `HTTPS` and `Https` are valid.
+* The `Url` parameter is required for each endpoint. The format for this parameter is the same as the top-level `Urls` configuration parameter except that it's limited to a single value.
+* These endpoints replace those defined in the top-level `Urls` configuration rather than adding to them. Endpoints defined in code via `Listen` are cumulative with the endpoints defined in the configuration section.
+* The `Certificate` section is optional. If the `Certificate` section isn't specified, the defaults defined in **Certificates** > **Default** are used. If no defaults are available, the server throws an exception and fails to start.
```

Comment:
It might be worth pointing out it will fallback to a dev certificate if available.